File: returned/python/valid_parentheses_test.py
from valid_parentheses import Solution

def test1():
    assert Solution().isValid("()") == True

def test2():
    assert Solution().isValid("()[]{}") == True

def test3():
    assert Solution().isValid("(]") == False

def test4():
    assert Solution().isValid("([])") == True
    
def test5():
    assert Solution().isValid("]") == False

File: csharp/LeetTests/Today1/ValidParenthesesTest.cs
using Today1.ValidParentheses;

namespace Today1;

internal class ValidParenthesesTest
{
    [Test]
    public void TestValidParentheses()
    {
        var solution = new Solution();
        Assert.That(solution.IsValid("()"), Is.EqualTo(true));
        Assert.That(solution.IsValid("()[]{}"), Is.EqualTo(true));
        Assert.That(solution.IsValid("(]"), Is.EqualTo(false));
        Assert.That(solution.IsValid("([])"), Is.EqualTo(true));
    }
}